MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Folder/FolderFormView.js

```diff
@@ -36,6 +32,22 @@ class FolderFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
     }
 
     /**
+     * Render a form
+     *
+     * @private
+     */
+    _renderForm() {
+        super._renderForm();
+        let title = $('#oo_folder_name', this.$el).val();
+        if (null === this._folderId) {
+            title = Translator.trans('open_orchestra_media_admin.table.folder.new');
+        }
+        $('#page-name', this.$el).html(title);
```

Comment:
@alavieille : all my answers in https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2150